Developer-Guide.md

```diff
@@ -27,23 +29,32 @@
 
 # Warning
 
-This document is written **specifically for developers**.
+This document is written **specifically for developers**: it is not intended for end users.
 
 # Assumptions
 
 - You are working on a non-critical test or development system.
-- You already have the following installed:
-  - [Docker](https://www.docker.com/).
-  - [golang](https://golang.org/dl) version 1.8.3 or newer.
-  - `make`.
-  - `gcc` (required for building the shim and runtime).
-
-- You have installed the Clear Containers `linux-container` and `qemu-lite`
-  packages containing the guest kernel images and hypervisor. These packages
-  are automatically installed when you install Clear Containers, but can be
-  installed separately as well:
-
-    https://github.com/clearcontainers/runtime/wiki/Installation
+
+# Initial setup
+
+The recommended way to create a development environment is to first install the
+packaged versions of the Kata Containers components to create a working
+system:
+
+  * [Fedora*](install/fedora-installation-guide.md)
```

Comment:
Will this create a mix of /usr/bin and /usr/local/bin on the system, though?  Should we just include directions on installing docker, qemu and VM artifacts?

Reply:
Yes - the official OBS binary for the runtime will be `/usr/bin/kata-runtime` and the developer version will be `/usr/local/bin/kata-runtime`. That's documented in the guide already (see https://github.com/kata-containers/documentation/blob/master/Developer-Guide.md#update-docker-configuration). We really don't want to encourage users to overwrite the packaged binaries, even on a dev box imho.